.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Missing newline at end of file

Reply:
Ahh, this is not needed, this is an artifact of my IDE, will remove this.